app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+import datetime
+from app.cafe import Cafe
```

Comment:
Separate your local imports and built-in library imports with one blank line according to PEP-8.